nextjs/hooks/polling.tsx

```diff
@@ -6,7 +6,7 @@ interface Props {
   error(): void;
 }
 
-const POLL_INTERVAL_IN_SECONDS = 15;
+const POLL_INTERVAL_IN_SECONDS = 999999;
```

Comment:
@emilos I increment the interval instead of removing this implementation

Reply:
smart :)